<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" alignment="Qt::AlignLeft">
    <widget class="QComboBox" name="coData">
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1" alignment="Qt::AlignLeft">
    <widget class="QComboBox" name="coParity">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mark</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Space</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Parity:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Stop bits:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" alignment="Qt::AlignLeft">
    <widget class="QComboBox" name="coStop">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Baud:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Porta:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Data bits:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="lePorta">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignLeft">
    <widget class="QComboBox" name="coBaud">
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btApply">
     <property name="text">
      <string>&amp;Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
